Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,258 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        private const string NotAvailable = "N/A";
```

Comment:
It goes both ways for this that people have told me.  In the past, people have said not to include punctuation, such as the period in this case, along with the constant word.  There reasoning was that "not available" is just "N/A" or "n/a" when we write it in the real world and so it should not be included.  The other side of the argument is that, because the period is being repeated with "N/A" then the period should be included to be "N/A.".  I have seen more people argue against including the period, even though I find it more annoying having the repeat it.  Because, the comment from previous reviewer suggested "N/A" (without any period), I left the period out.  I have included the period in the latest change.